ush/HWP_tools.py

```diff
@@ -7,6 +12,14 @@
 import fnmatch
 
 def check_restart_files(hourly_hwpdir, fcst_dates):
+    """Check the restart files.
+
+    Args:
+         hourly_hwpdir: Hourly ??? directory

```

Comment:
What is hourly_hwpdir?

Reply:
hourly_hwpdir is the directory where the RESTART data is copied (nwges/HOURLY_HWP)